pennylane/interfaces/batch/__init__.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage defines functions for interfacing devices with batch execution
+capabilities with different machine learning libraries.
+"""
+# pylint: disable=import-outside-toplevel)
+from functools import partial
+
+import pennylane as qml
+
+from .autograd import execute as execute_autograd
+
+
+def execute(tapes, device, gradient_fn, interface="autograd", accumulation="forward"):

```

Comment:
This is the `qml.interfaces.batch.execute` wrapper function/dispatcher.

It simply dispatches to the correct interfaces (for now, just autograd), as well as performing all shared validation.